src/core/linear_solver/src/method/4C_linear_solver_method_linalg.cpp

```diff
@@ -237,11 +237,15 @@ Teuchos::ParameterList translate_four_c_to_ifpack(const Teuchos::ParameterList&
 {
   Teuchos::ParameterList ifpacklist;
 
-  ifpacklist.set("fact: level-of-fill", inparams.get<int>("IFPACKGFILL"));
+  /*ifpacklist.set("fact: level-of-fill", inparams.get<int>("IFPACKGFILL"));
   ifpacklist.set("partitioner: overlap", inparams.get<int>("IFPACKOVERLAP"));
   ifpacklist.set("schwarz: combine mode",
       inparams.get<std::string>("IFPACKCOMBINE"));    // can be "Zero", "Add", "Insert"
   ifpacklist.set("schwarz: reordering type", "rcm");  // "rcm" or "metis" or "amd"
+  */
+
+  std::string xmlfile = inparams.get<std::string>("IFPACK_XML_FILE");
```

Comment:
```suggestion
  const std::string xmlfile = inparams.get<std::string>("IFPACK_XML_FILE");
```
... and also at other places throughout this PR.